datapreview.tex

```diff
@@ -34,6 +25,9 @@ \subsection{Data Preview 0}
 Data Preview 0 contains three stages, all based on simulated LSST-like data products. 
 The goals of DP0 are to serve as an early integration test of the LSST Science Pipelines and the Rubin Science Platform (RSP), and to enable a limited number of astronomers and students to begin early preparations for science with the LSST.
 
+The following sections outline which data products can be expected in each Data Preview and Data Release, and on what time scale.
+See Table~\ref{tab:milestones} in the Timeline section below for a combined view of the expected data preview schedule and its uncertainties.
```

Comment:
This reference is broken in the just-released version.

Reply:
Now fixed